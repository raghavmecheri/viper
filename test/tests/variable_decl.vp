int a = 10;
print(a);
int b = 20;
print(b);
int c;
c = 10;
print(c);
int d = c + 10;
print(d);

char e = 'a';
print(e);

bool x = true;
print(x);
bool y = false;
print(y);

float xx = 1.1;
print(xx);
float zz = 2.2;
print(zz);
float aa = xx + zz;
print(aa);
