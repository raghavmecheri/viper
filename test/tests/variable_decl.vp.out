10
20
10
20
a
1
0
1.1
2.2
3.3
